fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityInfoCard.kt

```diff
@@ -0,0 +1,186 @@
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.shopping.ui
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.ClickableSubstringLink
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.settings.SupportUtils
+import org.mozilla.fenix.shopping.store.ReviewQualityCheckState
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Info card UI containing an explanation of the review quality.
+ *
+ * @param modifier Modifier to apply to the layout.
+ * @param onLearnMoreClick Invoked when the user clicks to learn more about review grades.
+ */
+@Composable
+fun ReviewQualityInfoCard(
+    modifier: Modifier = Modifier,
+    onLearnMoreClick: (String) -> Unit,
+) {
+    ReviewQualityCheckExpandableCard(
+        title = stringResource(id = R.string.review_quality_check_info_title),
+        modifier = modifier,
+    ) {
+        ReviewQualityInfo(
+            modifier = Modifier.fillMaxWidth(),
+            onLearnMoreClick = onLearnMoreClick,
+        )
+    }
+}
+
+@Suppress("Deprecation")
+@Composable
+private fun ReviewQualityInfo(
+    modifier: Modifier = Modifier,
+    onLearnMoreClick: (String) -> Unit,
+) {
+    Column(
+        modifier = modifier,
+        verticalArrangement = Arrangement.spacedBy(24.dp),
+    ) {
+        // Any and all text formatting (bullets, inline substring bolding, etc.) will be handled as
+        // follow-up when the copy is finalized.
+        // Bug 1848219
+        Text(
+            text = stringResource(id = R.string.review_quality_check_info_overview),
+            color = FirefoxTheme.colors.textPrimary,
+            style = FirefoxTheme.typography.body2,
+        )
+
+        val link = stringResource(R.string.review_quality_check_info_learn_more_link)
+        val text = stringResource(R.string.review_quality_check_info_learn_more, link)
+        val linkStartIndex = text.indexOf(link)
+        val context = LocalContext.current
+        val linkEndIndex = linkStartIndex + link.length
+        ClickableSubstringLink(
```

Comment:
We're trying to move away from `ClickableSubstringLink` and move to `LinkText` for future clickable links in text blocks.

Reply:
Yeah, I think since it was officially `@Deprecated` it, there is a ticket to remove all of the usages, but not 100% that was made.